src/main/kotlin/org/rust/lang/core/resolve/ref/RsLitExprReferenceContributor.kt

```diff
@@ -190,3 +206,32 @@ private class RsLiteralWebReferenceProvider : PsiReferenceProvider() {
         private val SPACE_SPLITTER: Regex = Regex("\\s")
     }
 }
+
+/**
+ * Current support:
+ *
+ * [rust-lang](https://github.com/rust-lang/rust)
+ * - [RsInnerAttr] / [RsOuterAttr] - `unstable` - `issue`
+ * - [RsOuterAttr] - `rustc_const_unstable` - `issue`
+ */
+private class RsLiteralGitHubIssueProvider : RsLiteralWebReferenceProviderBase() {
+
+    override fun getReferencesByStringLiteral(expr: RsLitExpr, literal: RsLiteralKind.String): Array<PsiReference> {
+        if (expr.containingCrate.origin != PackageOrigin.STDLIB) return PsiReference.EMPTY_ARRAY
+
+        val exprMetaItem = expr.parent as? RsMetaItem ?: return PsiReference.EMPTY_ARRAY
+
+        if (exprMetaItem.name != "issue") return PsiReference.EMPTY_ARRAY
+
+        val issueNumber = literal.value?.toLongOrNull() ?: return PsiReference.EMPTY_ARRAY
+        val attr = exprMetaItem.parentOfType<RsAttr>() ?: return PsiReference.EMPTY_ARRAY
+        val attrName = attr.metaItem.name
+
+        return when {
+            attr is RsOuterAttr && attrName == "rustc_const_unstable" || attrName == "unstable" ->
```

Comment:
Are you sure that we need `attr is RsOuterAttr` check at all?
I'd drop this check not to make implementation more complicated and extract all attributes that we support here into a list in the companion object.
It allows us easily to support other attributes if needed by modification attribute list